test/Font.spec.ts

```diff
@@ -16,4 +16,14 @@ context("Font", () => {
     assert.strictEqual(Buffer.from(os.buffer).toString("base64"), fontB64);
   });
 
+  it("findGlyphByUnicode", () => {
```

Comment:
It is also nice to have a test of how findGlyphByUnicode behaves in case the required format missed and the requestable glyph is missing. So it's clear for the lib user what it should expect, an error or just null as a result without digging into the code. This also will help maintain the library and prevent unwanted behavior changes at the edge cases.

Reply:
fixed